tutorials/scripting/c_sharp/c_sharp_differences.rst

```diff
@@ -486,22 +499,52 @@ Use the ``Godot.Collections.Dictionary<T>(Godot.Collections.Dictionary)`` constr
 Variant
 -------
 
-``System.Object`` (``object``) is used instead of ``Variant``.
+``Godot.Variant`` is used as the variant type, any variant-compatible type can be converted from/to it.
+Consider avoiding ``Godot.Variant`` and take advantage of C#'s type safety when possible,
+only use ``Godot.Variant`` when necessary to use Godot untyped APIs.
+
+The ``Variant.As{TYPE}`` methods or the generic ``Variant.As<T>`` method can be used to convert
+a ``Godot.Variant`` to a C# type. Since the ``Godot.Variant`` type contains implicit conversions
+defined for all the supported types calling these methods directly is usually not necessary.
+
+Use the ``CreateFrom`` method overloads or the generic ``From<T>`` method to convert a C# type
+to a ``Godot.Variant``.
 
 Communicating with other scripting languages
 --------------------------------------------
 
 This is explained extensively in :ref:`doc_cross_language_scripting`.
 
-.. _doc_c_sharp_differences_yield:
+.. _doc_c_sharp_differences_await:
 
-Yield
------
+Await keyword
+-------------
+
+Something similar to GDScript's ``await`` keyword can be achieved with C#'s
+`await keyword <https://docs.microsoft.com/en-US/dotnet/csharp/language-reference/keywords/await>`_.
```

Comment:
```suggestion
Something similar to the GDScript's ``await`` keyword can be achieved with the C#'s
`await keyword <https://docs.microsoft.com/en-US/dotnet/csharp/language-reference/keywords/await>`_.
```

Reply:
For some reason it sounds weird to me to use the `the` article with `'s`. But if you think it's more correct like this.

Reply:
I guess if it's apostrophe'd, then we don't need an article. You can ignore this.